portfolio/src/main/java/com/google/sps/servlets/DeleteMessageServlet.java

```diff
@@ -31,7 +29,13 @@
 public class DeleteMessageServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    long id = Long.parseLong(request.getParameter("id"));
+    long id;
+    try {
+       id = Long.parseLong(request.getParameter("id"));
+    } catch (NumberFormatException e) {
+      System.err.println("Message id for deleting is not long: " + e);
+      return;
+    }
```

Comment:
Nothing. What should be? :)

Reply:
> I am not sure what happened with my comment on some PR before my vacation about HTTP status codes.
That's on me -- we decided to move ahead with the old PR to unblock the progress for last week because you were not available for clarification :) 
Thanks for the API guidelines, very useful resource! 
@LadyPython, could you check on how to integrate an appropriate response code (here and in the other cases)?